dlk/python/dlk/core/optimizer.py

```diff
@@ -446,23 +376,50 @@ def pass_pack_weights(graph: Graph) -> None:
 
         # Quantize the weights
         weight_quantizer.run_forward()
+
+        def pad_to_multiple_of_b(tensor, axis, b):
+            shape = list(tensor.shape)
+            pad = (((shape[axis] + b - 1) // b) * b) - shape[axis]
+            shape[axis] = pad
+            return np.zeros(shape) if pad else None
+
+        padded_data = np.copy(weight_quantizer.data)
+
+        for axis in [0, 3]:
+            pad_tensor = pad_to_multiple_of_b(padded_data, axis, b)
+            if pad_tensor is not None:
+                padded_data = np.append(padded_data, pad_tensor, axis=axis)
+
+        tca_output = np.copy(padded_data)
+        oc, kh, kw, kd = padded_data.shape[:]
+        padded_data = padded_data.flatten()
```

Comment:
Why did you flatten padded_data?

Reply:
Because I wanted to access with indices to the array as in here:
```
... = padded_data[idx]
```

Reply:
It seems that padded_data[g * b + d, o, h, p * b + d] (padded_data is not-flatten) is ok.

Reply:
Ah, yes, it is equivalent. For me was easier to keep it same than we used to do in the runtime.

Reply:
OK.